Sources/Sentry/Public/SentryReplay.h

```diff
@@ -0,0 +1,50 @@
+#import <Foundation/Foundation.h>
+
+#if __has_include(<Sentry/SentryDefines.h>)
+#    import <Sentry/SentryDefines.h>
+#else
+#    import <SentryWithoutUIKit/SentryDefines.h>
+#endif
+
+#if SENTRY_TARGET_REPLAY_SUPPORTED
+
+@class UIView;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryReplay : NSObject
```

Comment:
also, dunno if you want to be consistent with naming, but metrics is called `SentryMetricsAPI`. For me it's fine either way though